### PR TITLE
feature: Install webpack

### PR DESCRIPTION
**Pull request by anastasiya-nd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,13 +19,23 @@
   },
   "homepage": "https://github.com/anastasiya-nd/budget#readme",
   "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/preset-env": "^7.15.0",
+    "@babel/preset-react": "^7.14.5",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^6.2.0",
```
Comment (ok2ju) [inline package.json:26]: You don't need `css-loader`. You'll use `styled-components`

File: package.json [hunk 2]
```diff
@@ -17,13 +19,23 @@
...
     "editor": "^1.0.0",
```
Comment (ok2ju) [inline package.json:27]: What for this package?

File: package.json [hunk 3]
```diff
@@ -17,13 +19,23 @@
...
     "eslint": "^7.32.0",
     "eslint-config-standard": "^16.0.3",
```
Comment (ok2ju) [inline package.json:29]: Let's use `airbnb` code style instead of `standard`

File: package.json [hunk 4]
```diff
@@ -17,13 +19,23 @@
...
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.0",
-    "eslint-plugin-react": "^7.24.0"
+    "eslint-plugin-react": "^7.24.0",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
```
Comment (ok2ju) [inline package.json:35]: You don't need `style-loader` as you will use `styled-components`

File: package.json [hunk 5]
```diff
@@ -17,13 +19,23 @@
   },
   "homepage": "https://github.com/anastasiya-nd/budget#readme",
   "devDependencies": {
+    "@babel/core": "^7.15.0",
```
Comment (ok2ju) [inline package.json:22]: Where is `.babelrc` file?

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  mode: 'development',
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader'
+      },
+      {
+        test: /\.(sass|css)$/,
```
Comment (ok2ju) [inline webpack.config.js:13]: You don't need this loader

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+        use: ['style-loader', 'css-loader']
+      },
+      {
+        test: /\.ttf$/,
```
Comment (ok2ju) [inline webpack.config.js:17]: You don't need this loader for now

**Review by ok2ju — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (anastasiya-nd) [reply package.json:27]: I don't understand what you are saying.

Comment (ok2ju) [reply package.json:27]: Why did you install this package (`editor`)?

Comment (anastasiya-nd) [reply package.json:27]: I thought I had to use eslint and editor together

File: package.json (hunk 3 above)
Comment (anastasiya-nd) [reply package.json]: I have to remove eslint and install to main

File: package.json (hunk 2 above)
Comment (ok2ju) [reply package.json:27]: What is editor?

**Review by ok2ju — CHANGES_REQUESTED** [review]

**Pull request closed without merging**